app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -340,11 +340,11 @@ class MainActivity : AppCompatActivity(),
 
         // initialize the text & HTML
         if (!isRunningTest) {
-            aztec.sourceEditor.displayStyledAndFormattedHtml(EXAMPLE)
+            aztec.sourceEditor?.displayStyledAndFormattedHtml(EXAMPLE)
         }
 
         if (savedInstanceState == null) {
-            aztec.visualEditor.fromHtml(aztec.sourceEditor.getPureHtml())
+            aztec.visualEditor.fromHtml(aztec.sourceEditor?.getPureHtml()!!)
```

Comment:
thanks for the explanation! Learning a lot from your reviews 🙇 
What would we need to do, if we wanted to do something about it?